misc/msr-lock.md

```diff
@@ -25,60 +23,96 @@ So to fix it we have 2 options:
   
 Note: Penyrn based machines actually don't need to worry about unlocking this register
 
+## Checking if your firmware supports CFG Lock unlocking
+
+Before proceeding with the reading of this guide, first of all you'd need to check if your firmware supports CFG Lock unlocking.
+To check it, you can proceed into two ways:
+
+1. use the DEBUG version of OpenCore and check what the log says about CFG Lock
+2. use a tool called `VerifyMsrE2` which will speed up the whole checking process
+
+Personally I prefer using the first choice, but to avoid confusion I'll choose option 2.
+Add the tool inside `EFI/OC/Tools` and `config.plist` then run `VerifyMsrE2.efi` and check if you read 
+
+```
+This firmware has LOCKED MSR 0xE2 register!
+```
+
+If you read
+
+```
+This firmware has UNLOCKED MSR 0xE2 register!
+```
+
+you don't need to do anything and you can disable `Kernel -> Quirks -> AppleCpuPmCfgLock` and `Kernel -> Quirks -> AppleXcpmCfgLock`.
+
 ## Disabling CFG Lock
 
-So you've installed macOS but you're using those pesky `CFG-Lock` patches that we want to get rid of, well to do this is fairly simple. You'll need the following:
+So you've created the EFI folder but you can't still boot without unlocking before CFG Lock. In order to do this you'll need the following:
 
-Inside your EFI/OC/Tools folder and config.plist:
+Inside your `EFI/OC/Tools folder` and `config.plist`:
 
-* [VerifyMsrE2](https://github.com/acidanthera/OpenCorePkg/releases)
 * [Modified GRUB Shell](https://github.com/datasone/grub-mod-setup_var/releases)
 
 And some apps to help us out:
 
-* [UEFITool](https://github.com/LongSoft/UEFITool/releases) (Make sure it's UEFITool and not UEFIExtrac)
+* [UEFITool](https://github.com/LongSoft/UEFITool/releases) (Make sure it's UEFITool and not UEFIExtract)
 * [Universal-IFR-Extractor](https://github.com/LongSoft/Universal-IFR-Extractor/releases)
 
-And don't forget to disable the following from your config.plist under `Kernel -> Quirks`:
-
-* `AppleCpuPmCfgLock`
-* `AppleXcpmCfgLock`
-
 And the final part, grabbing your BIOS from the vendors' website.
 
 Now the fun part!
 
-## Checking if CFG-Lock can be turned off
 
-Boot OpenCore and select the `VerifyMsrE2` option in the picker. This tool will tell you whether your BIOS supports CFG-Lock and if it can be unlocked.
 
 ## Turning off CFG-Lock manually
 
+##### Please note that the only firmwares that can be directly opened by UEFITool are ASUS, MSI and ASRock. Other firmwares need a special procedure which we'll not directly cover into this guide. For Dell firmwares please refer to [dreamwhite's guide](https://github.com/dreamwhite/bios-extraction-guide/tree/master/Dell)
```

Comment:
I think instead of using a header, `**` may be more ideal

Reply:
Alright, I'll edit as suggested